core/src/test/java/org/elasticsearch/index/query/BaseQueryTestCase.java

```diff
@@ -218,8 +218,8 @@ public void testFromXContent() throws IOException {
 
         QueryBuilder newQuery = queryParserService.queryParser(testQuery.getName()).fromXContent(context);
         assertNotSame(newQuery, testQuery);
-        assertEquals("Queries should be equal", testQuery, newQuery);
-        assertEquals("Queries should have equal hashcodes", testQuery.hashCode(), newQuery.hashCode());
+        assertEquals("Queries should be equal: " + testQuery + " vs. " + newQuery, testQuery, newQuery);
```

Comment:
I think I removed this testQuery vs. newQuery. when it fails I see the same message twice:
testQuery vs. newQuery and then
expected: testQuery, actual: newQuery

I don't see the added value of this, maybe it's my IDE or something?